### DISCUSSION TITLE
Route tun device traffic through tun2socks

### DISCUSSION THREAD

**Discussion by armantc** [xjasonlyu/tun2socks#223] [General]
hi,
i have setuped softether vpn server on vps, that use tap device, and have local dhcp server get ip to each client,
tap device ip is  192.168.24.0   subnet 192.168.24.0/21  by default use eth0 as gateway, i want to internet traffic of tun0 include udp and tcp goes through tun2socks, like below

tun0 =>  tun2socks => local socks + http + tls  => eth0 ====> remote server

**Comment by liuzhioic** [comment]
me too.
